setup.py

```diff
@@ -151,7 +151,7 @@ def get_git_sha() -> str:
             "sqlalchemy-databricks>=0.2.0",
         ],
         "db2": ["ibm-db-sa>=0.3.5, <0.4"],
-        "dremio": ["sqlalchemy-dremio>=1.1.5, <1.3"],
+        "dremio": ["sqlalchemy-dremio>=1.2.1, <4"],
```

Comment:
Is this too wide of a range?

Reply:
@rusackas if we drop its odbc support, we could limit to `>=3, <4`.